deployment/start_and_deploy_to_server.sh

```diff
@@ -71,7 +71,7 @@ if [[ $RESET_BACKEND_DATABASE_AND_REPOPULATE == true ]]; then
 fi
 
 echo "Starting docker compose stack."
-ssh ubuntu@"$target_server_url" "cd $location; sudo docker-compose pull; sudo docker-compose --profile $profile up -d --build"
+ssh ubuntu@"$target_server_url" "cd $location; sudo docker compose pull; sudo docker compose --profile $profile up -d --build"
```

Comment:
Why is it better to use "docker compose" instead of "docker-compose" on our servers?
I think at least on dev1 I installed "docker-compose" too.

Reply:
It is only a forwarding. Dev1 does not actually use docker-compose but docker compose.

Reply:
Habe ich nicht ganz verstanden.   Können wir vllt später besprechen, wenn ich durch bin mit dem Review.